commands/cluster-slots.md

```diff
@@ -41,29 +43,36 @@ slot range reply.
    3) 1) "127.0.0.1"
       2) (integer) 30001
       3) "09dbe9720cda62f7865eabc5fd8857c5d2678366"
+      4) "host-1.redis.domain.com"
```

Comment:
`domain.com` is an actual website. It looks like a registrar.

Better use a domain dedicated to examples such as `example.com`.